frontend/src/pages/manage-member/AddMember.jsx

```diff
@@ -0,0 +1,262 @@
+import React, { useState, useEffect } from "react";
+import "./styles.scss";
+import {
+  Row,
+  Col,
+  Button,
+  Space,
+  Modal,
+  Form,
+  Input,
+  Select,
+  Typography,
+  notification,
+} from "antd";
+import { api, store } from "../../lib";
+
+const { Title } = Typography;
+const { Option } = Select;
+
+const AddMember = ({
+  isAddMemberVisible,
+  setIsAddMemberVisible,
+  setReload,
+  reload,
+  selectedMember,
+  setSelectedMember,
+}) => {
+  const [sending, setSending] = useState(false);
+  const [form] = Form.useForm();
+  const { optionValues } = store.useState((s) => s);
+  const { member_type, isco_type } = optionValues;
+  const iscoType = isco_type.filter((isco) => isco.name !== "All");
+
+  const isAdd = !selectedMember;
+  const disableFields = selectedMember !== null;
+  const requiredFields = isAdd ? true : false;
+  const modalTitle = isAdd ? "New Member" : "Update Member";
+  const buttonOkText = isAdd ? "Add Member" : "Update Member";
+
+  // set initial form values
+  useEffect(() => {
+    if (!selectedMember) {
+      form.setFieldsValue({ first_name: "" });
+      form.setFieldsValue({ last_name: "" });
+      form.setFieldsValue({ member_type: [] });
+      form.setFieldsValue({ isco_type: [] });
+    }
+    if (selectedMember?.id) {
+      const { name, member_type, isco_type } = selectedMember;
+      const first_name = name.split(" ")[0];
+      const last_name = name
+        .split(" ")
+        .filter((n, i) => i !== 0)
+        .join(" ");
+      form.setFieldsValue({ first_name: first_name });
+      form.setFieldsValue({ last_name: last_name });
+      form.setFieldsValue({ member_type: member_type });
+      form.setFieldsValue({ isco_type: isco_type });
+    }
+  }, [selectedMember]);
+
+  const handleOnClickModalCancel = () => {
+    form.resetFields();
+    setIsAddMemberVisible(false);
+    setSelectedMember(null);
+  };
+
+  const onFinish = (values) => {
+    const { first_name, last_name, member_type, isco_type } = values;
+    const payload = {
+      code: "1",
+      name: `${first_name} ${last_name}`,
+      active: true,
+      member_type: member_type,
+      isco_type: isco_type,
+    };
+    setSending(true);
+    if (!selectedMember) {
+      api
+        .post("/organisation", payload)
+        .then(() => {
+          notification.success({
+            message: "Member has been successfully added",
```

Comment:
Move this message into the `description` prop, and replace the message with "Success"